meta/kwoptarg.rb

```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```

Comment:
Ok, I lied. I forgot you have to do special things to run `mutest` on itself (at least, I assume that's the problem). Otherwise we have a bug.